src/backend/tests/unit/test_database.py

```diff
@@ -67,8 +67,30 @@ async def test_read_flows(client: TestClient, json_flow: str, active_user, logge
     assert len(response.json()) > 0
 
 
+async def test_read_flows_pagination(client: TestClient, json_flow: str, active_user, logged_in_headers):
+    response = await client.get("api/v1/flows/", headers=logged_in_headers)
+    assert response.status_code == 200
+    assert response.json()["page"] == 1
+    assert response.json()["size"] == 50
+    assert response.json()["pages"] == 0
```

Comment:
This is a bit unintuitive. How can we be on a page, but there be 0 pages?

Reply:
Its because is an empty array, so there is going to be no pages at all

Reply:
As in, why is `response.json()["page"] == 1`, but `pages` is 0? I would have expected the response to say "Sorry, there are no pages, so we can't give you page 1 (or page 3, etc)"

I assume this is just how the fastapi pagination library behaves, but just curious as to why. 

Can we add a test when there _are_ items, and make sure that the correct number of items is returned per page and we can iterate over the pages to retrieve the total?